front-end/src/components/Post/GovernanceSideBar/Referenda/VoteReferendum.tsx

```diff
@@ -141,7 +141,6 @@ const VoteRefrendum = ({ className, referendumId, address, accounts, onAccountCh
 						/>
 						<VoteLock/>
 						<AyeNayButtons
-							disabled={!isApiReady}
```

Comment:
For referendum I certainly think it makes sense to disable them until the api is there.